openapi.yaml

```diff
@@ -3682,8 +3693,7 @@ components:
                     type: array
                     items:
                         $ref: "#/components/schemas/RequestConfiguration"
-                    x-speakeasy-name-override: request_configuration_list_data
-                    x-speakeasy-param-suppress-computed-diff: true
+                    x-speakeasy-ignore: true
```

Comment:
`request_configuration_list` is deprecated and people should use `request_configurations`